pkg/config/config.go

```diff
@@ -147,6 +147,22 @@ type Homebrew struct {
 	Goarm                 string               `yaml:"goarm,omitempty"`
 }
 
+// Krew contains the krew section.
+type Krew struct {
+	IDs                   []string     `yaml:"ids,omitempty"`
+	Name                  string       `yaml:"name,omitempty"`
+	Index                 RepoRef      `yaml:"index,omitempty"`
+	CommitAuthor          CommitAuthor `yaml:"commit_author,omitempty"`
+	CommitMessageTemplate string       `yaml:"commit_msg_template,omitempty"`
+	Caveats               string       `yaml:"caveats,omitempty"`
+	ShortDescription      string       `yaml:"short_description,omitempty"`
+	Description           string       `yaml:"description,omitempty"`
```

Comment:
FWIW: we expect the author to line-wrap this at 80 chars as a multi-line YAML string (we provide this feedback during the code-review in krew-index) but it's not a hard requirement especially when URLs are in the description.